pkg/providers/instance/instance.go

```diff
@@ -653,7 +683,11 @@ func GetAllSingleValuedRequirementLabels(instanceType *corecloudprovider.Instanc
 }
 
 // pick the "best" SKU, priority and zone, from InstanceType options (and their offerings) in the request
-func (p *DefaultProvider) pickSkuSizePriorityAndZone(ctx context.Context, nodeClaim *karpv1.NodeClaim, instanceTypes []*corecloudprovider.InstanceType) (*corecloudprovider.InstanceType, string, string) {
+func (p *DefaultProvider) pickSkuSizePriorityAndZone(
+	ctx context.Context,
+	nodeClaim *karpv1.NodeClaim,
+	instanceTypes []*corecloudprovider.InstanceType,
+) (*corecloudprovider.InstanceType, string, string) {
```

Comment:
iirc alex doesnt' like this formatting? Maybe im mistaken I feel like i tried to do it a long time ago and got told not to, if we are gonna go this route as well, maybe a linter for line length is in order? 

Thoughts?

Reply:
I _think_ general guidance is that the function definition should either be:
1. On a single line
2. Like this.

I don't really want to enforce a line-length limit, I'm fine with some long lines, but at least IMO for functions like `getLaunchTemplate` above (which was already split across multiple lines) this format is objectively easier to read.

Some of the error messages above that I reformatted were long enough that they were off the right hand side of my monitor (which isn't ultra-wide, but it's not tiny either).

Reply:
Can you point to a case?